database/migrations/2023_08_20_014254_update_event_registration_add_remarks.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up() {
+        Schema::table('event_registration', function ($table) {
+            $table->string('remarks', 1024)->nullable();
```

Comment:
Why 1024 length? FYI - there is no protection in the form to guard from longer length input - causes sign-up to fail.

Reply:
Arbitrary choice - typically 1 MiB is a good size limit for "any other information?" text fields - past that, you should just write an email directly to the EC.